docs/topics/note-preview-booster-source.adoc

```diff
@@ -1 +1,2 @@
-NOTE: To view the source code and README file of this Booster, link:{link-getting-started-guide}#oso-create-booster[download and extract the Booster ZIP file].
+NOTE: To view the source code and README file of this booster, download and extract the ZIP file with the booster.
+To get the download link of the ZIP file, build the booster locally following the instructions in the link:{link-getting-started-guide}#oso-create-booster[Using OpenShift to Create a Booster] chapter of the {getting-started-guide-name}.
```

Comment:
You don't really need to "build the booster locally", right? You just need to "create the booster using the .zip flow"? But I'm missing the context here, so I might be easily wrong. These small files are nice for composability, but they lose all the context.

Reply:
Ah, very well. I will reword.

Re context: I agree, the module files generally shouldn't be this small.